zstacklib/zstacklib/utils/gpu.py

```
@@ -228,3 +229,18 @@ def get_huawei_gpu_product_name_cmd(npu_id, iswindows=False):
 def reload_hygon_gpu_driver_cmd():
     cmd = "hy-smi --unloaddriver && hy-smi --loaddriver"
     return cmd
+
+
+def get_vastai_type():
+    r, o, e = bash_roe("lspci | grep Vastai")
```

Ruff (0.11.9)

`bash_roe` may be undefined, or defined from star imports

(F405)